guillotina_batch/api.py

```diff
@@ -215,5 +217,5 @@ class Batch(Service):
     async def __call__(self):
         results = []
         for message in await self.request.json():
-            results.append(await self.handle(message))
+            results.append(await self.handle(message, self.request.headers))
```

Comment:
`self.request` is a member, we do not need to pass in the headers value here--we have access to it in the handle method.